pydicom/data/utils.py

```diff
@@ -0,0 +1,44 @@
+"""pydicom data manager utils"""
+#
+# Copyright (c) 2008-2012 Darcy Mason
+# Copyright (c) 2017 pydicom AUTHORS
+# This file is part of pydicom, released under a modified MIT license.
+#    See the file LICENSE included with this distribution, also
+#    available at https://github.com/pydicom/pydicom
+#
+
+import os
+import fnmatch
+
+
+def recursive_find(base, pattern=None):
+    '''recursively find files based on a pattern
+    '''
+    if pattern is None:
+        pattern = "*"
+    files = []
+    for root, dirnames, filenames in os.walk(base):
+        for filename in fnmatch.filter(filenames, pattern):
+            files.append(os.path.join(root, filename))
+
+    return files
+
+
+def get_files(bases, pattern=None):
```

Comment:
Do we need to function `get_files` and `recursive_find`. Is `recursive_find` used elsewhere?